calico_containers/pycalico/netns.py

```diff
@@ -180,6 +179,32 @@ def remove_ip_from_ns_veth(container_pid, ip, veth_name_ns):
                        "dev", "%(device)s" % veth_name_ns])
 
 
+def provision_veth(self, endpoint, ns_pid, veth_name_ns):
```

Comment:
For rkt, we'll need this method to take `ns_path` instead of `ns_pid`. That will mean rewiring a few functions in the netns.py module (including the NamedNamespace class). I suggest creating a get_docker_ns_path(pid) helper function to encapsulate the pid=>path logic for the docker case. We get the full path in the rkt case, so we don't need a helper method there.